.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: node_js
 
-before_script:
- - sh -e /etc/init.d/xvfb start
-
-install: ant
-
 env:
- - DISPLAY=':99.0'
+  global:
+    - DISPLAY=':99.0'
+  matrix:
+    - MOZDOWNLOADPARAMS='-t daily'
+    - MOZDOWNLOADPARAMS='-t daily --branch=mozilla-aurora'
+    - MOZDOWNLOADPARAMS='-t release -v latest-beta'
+    - MOZDOWNLOADPARAMS='-t release -v latest'
+    - MOZDOWNLOADPARAMS='-t release -v latest-esr'
```

Comment:
I see. Maybe I misremember that ESR31 was constantly failing while others have intermittent orange. In that case its fine for me.